File: Programmers/LV.1/음양 더하기.py
def solution(absolutes, signs):
    ans = 0
    for i in range(len(signs)):
        ans = ans + absolutes[i] if signs[i] else ans - absolutes[i]
    return ans

##############
    # ans = 0
    #
    # for i in range(len(signs)):
    #     if signs[i]:  # true 면
    #         ans += absolutes[i]
    #     else:
    #         ans -= absolutes[i]
    #
    # return ans

File: Programmers/LV.1/음양더하기.js
const absolutes = [4, 7, 12];
const signs = [true, false, true];

function solution(absolutes, signs) {
  var answer = 0;
  for (let i = 0; i < absolutes.length; i++) {
    answer = signs[i] ? answer + absolutes[i] : answer - absolutes[i];
    console.log(answer);
  }
  return answer;
}

solution(absolutes, signs);
